sample-input/sph-factors/assembly/compute-mgxs.py

```diff
@@ -0,0 +1,58 @@
+import copy
+import numpy as np
+import openmc.mgxs
+from infermc.materialize import differentiate_mgxs_lib
```

Comment:
The materialize module seems to only be in the `sph` branch of your fork of InferMC. Could this branch be pushed to the master repo or merged into the master repo's develop branch? In case someone tries to pick this up down the road and sees the infermc dependency, it would be nice if this capability comes with the repo they are likely to clone to avoid confusion.